__tests__/app.test.js

```diff
@@ -133,3 +133,48 @@ describe("\nGET /api/reviews/:review_id\n", () => {
       .then((response) => expect(response.body.msg).toBe("invalid path"));
   });
 });
+describe("\nGET /api/reviews/:review_id/comments\n", () => {
+  test("GET 200 status and matching comments", () => {
+    const review_id = 3;
+    return request(app)
+      .get(`/api/reviews/${review_id}/comments`)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments).toEqual([
```

Comment:
Should only include the keys as per the ticket:
comment_id, votes, created_at, author, body

Could also use a forEach loop and jest matchers to check data types for each property rather than an exact match at each index.